README.md

```diff
@@ -28,7 +28,9 @@ The `proj` prefix is short for "projection", and is not a reference to the PROJ/
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
-| proj:epsg        | integer\|null   | **REQUIRED.** [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:code        | integer\|null   | **REQUIRED.** [EPSG code](http://www.epsg-registry.org/) or other code (e.g., [IAU](http://voparis-vespa-crs.obspm.fr:8080/web/2015.html)) of the datasource |
```

Comment:
don't allow these to be null

Reply:
Updated. Also updated the proj:epsg - that was an integer. Will that change have unintended side effects?